### ISSUE TITLE
Great Job! Would the "Clean" training set (i.e., after removing the identified harmful sample) you used to re-train the VLLMs be released? 

### ISSUE THREAD

**Issue by pzs19** [ys-zong/VLGuard#6]

**Comment by ys-zong** [comment]
Yes, I'll aim to release it soon. At the same time, if you want to reproduce it yourself, you can take the LlamaGuard model to iterate over the whole dataset and filter the samples labelled as "unsafe".

**Comment by youngwanLEE** [comment]
@ys-zong Hi

Could you provide the cleaned training JSON file? It would be very helpful if you share it.

Thanks in advance.

**Comment by ys-zong** [comment]
Sorry for the long wait. It's uploaded [here](https://huggingface.co/datasets/ys-zong/VLGuard/blob/main/llava_v1_5_mix_clean.json) now. It's in the exact format as LLaVA-v1.5 training json, but pls note that it was filtered by an external model LlamaGuard, and therefore there might be false negatives.
